shopcart.py

```diff
@@ -237,6 +237,19 @@ def create_products(sid):
         return make_response(jsonify(message), rc)
 
 ######################################################################
+# DELETE A PRODUCT FROM A SHOPPING CART
+######################################################################
+@app.route('/shopcarts/<int:sid>/products/<int:sku>', methods=['DELETE'])
+def delete_products(sid,sku):
+    for i in range(len(shopping_carts)):
+    	if shopping_carts[i]['sid'] == sid:
+    	    for j in range(len(shopping_carts[i]['products'])):
+    	    	if shopping_carts[i]['products'][j]['sku'] == sku:
+    	    		del shopping_carts[i]['products'][j]
+    	    		break
+    return '', HTTP_204_NO_CONTENT
+    
```

Comment:
I think we should show appropriate error messages of cart not present and product not present if user enters invalid sid or sku. What do you think?